packages/components/src/components/Combobox/ComboboxPopover/ComboboxPopover.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { forwardRef } from "react";
+import {
+  ComboboxPopover as AriakitComboboxPopover,
+  ComboboxPopoverProps as AriakitComboboxPopoverProps,
+} from "@ariakit/react";
+import { Box } from "../../../primitives/Box";
+
+export const ComboboxPopover = forwardRef<
+  HTMLDivElement,
+  AriakitComboboxPopoverProps & {
+    children: React.ReactNode;
+  }
+>(({ children, ...props }, ref) => {
+  return (
+    <Box.div
+      as={AriakitComboboxPopover}
+      backgroundColor="bgPrimary"
+      borderRadius="borderRadius20"
+      boxShadow="shadowStrong"
+      display="flex"
+      flexDirection="column"
+      gutter={3}
+      maxHeight="250px"
+      outlineColor={{ focus: "colorBorderPrimary" }}
+      outlineStyle={{ focus: "solid" }}
+      outlineWidth={{ focus: "borderWidth10" }}
+      overflow="auto"
+      overscrollBehavior="contain"
+      portal
+      ref={ref}
+      sameWidth
+      unmountOnHide
+      zIndex="zIndex50"
```

Comment:
why 50 ?

Reply:
> why 50 ? 
Sorry, I just used what we had on the talent-web. 

Following what Rafa suggested, I added a `zIndex` prop with `zIndex50` as default.